### ISSUE TITLE
Nothing for font-variant-east-asian CSS property

### ISSUE THREAD

**Issue by AlbertWiersch** [Fyrd/caniuse#5524]

**Comment by Schweinepriester** [comment]
Exists on MDN and in BCD:
- https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-east-asian
- https://github.com/mdn/browser-compat-data/blob/master/css/properties/font-variant-east-asian.json

Probably just needs to be enabled?

**Comment by AlbertWiersch** [comment]
Yes, seems like it. I'm new here and don't know how to do that... I assume someone with the appropriate access needs to do it? Thanks.

**Comment by MikeSummers** [comment]
https://caniuse.com/?search=font-variant-east-asian has 0 results found
https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-east-asian has all the main browsers at full support except ie

**Comment by svgeesus** [comment]
The [test results for CSS Fonts 3](https://test.csswg.org/harness/results/css-fonts-3_dev/grouped/section/6.8/) show support in Blink, Gecko and Webkit. Clicking on results shows the specifics of browser versions tested. At the time, Edge was using it's own engine and did not support it (Edge now uses the Chromium blink engine, and so does support it) and Trident (MS IE) never supported it.

Drilling down [on Chrome results](https://test.csswg.org/harness/details/css-fonts-3_dev/font-variant-east-asian-02/engine/blink/) support starts with **Chrome 63**. Similarly [Gecko](https://test.csswg.org/harness/details/css-fonts-3_dev/font-variant-east-asian-02/engine/gecko/) from **at least Gecko 54** (there isn't a failing test recorded for Gecko, so finding the precise version would entail dredging through release notes) and for [Webkit](https://test.csswg.org/harness/details/css-fonts-3_dev/font-variant-east-asian-03/engine/webkit/) it was **Safari 11.2**
